components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -719,149 +718,20 @@ <h1 class="can-collapse defclose" data-name="details">
         
     <!-- ANNOTATIONS -->
 
+        <!-- templates are loaded based on config -->
+        {% for t in ome.metadata_pane|dictsort:"index" %}
```

Comment:
Right, if you don't have indexes them spaced out, 10, 20, 30, etc, then you can't insert one.
I guess it's just a decision as to what's the nicest default indexes to use. E.g can you insert ```"1.1"``` into ```["1", "2"]```?